src/config.rs

```diff
@@ -56,22 +62,50 @@ impl Config {
     }
 }
 
-fn default_issuer() -> String {
-    DEFAULT_ISSUER.to_string()
-}
-
-#[derive(Debug, Deserialize, Getters, Clone)]
+#[derive(Debug, Deserialize, Getters)]
 pub struct Audience {
     name: String,
     permissions: Vec<String>,
 }
 
-#[derive(Debug, Deserialize, Getters, Clone)]
+#[derive(Debug, Deserialize, Getters)]
 pub struct User {
     name: String,
     permissions: Vec<String>,
 }
 
+#[derive(Debug, Deserialize, Getters)]
+pub struct UserInfo {
+    #[serde(default = "defaults::user_info_name")]
+    name: String,
+    #[serde(default = "defaults::user_info_given_name")]
+    given_name: String,
+    #[serde(default = "defaults::user_info_family_name")]
+    family_name: String,
+    #[serde(default = "defaults::user_info_gender")]
+    gender: String,
+    #[serde(default = "defaults::user_info_birthdate")]
+    birthdate: String,
+    #[serde(default = "defaults::user_info_email")]
+    email: String,
+    #[serde(default = "defaults::user_info_picture")]
+    picture: String,
```

Comment:
on auth0 staging (and I think also on production) the id_token includes also the roles in a field called "https://prima.it/roles". This field has been added using a custom action.

By calling "/userinfo" API this field seems to not be returned instead.